lib/auth/keystore/aws_kms.go

```diff
@@ -56,21 +56,28 @@ const (
 
 	pendingKeyBaseRetryInterval = time.Second / 2
 	pendingKeyMaxRetryInterval  = 4 * time.Second
-	pendingKeyTimeout           = 30 * time.Second
+	// TODO(dboslee): waiting on AWS support to answer question regarding
+	// long time for GetPublicKey to succeed after updating key via UpdatePrimaryRegion.
```

Comment:
> KMS service team will deploy a fix by May 23 to resolve the GetPublicKey operation failures that occur when keys are in the 'updating' state. The service team has confirmed the code update is ready for deployment.

Update from AWS. Going to leave this until I am able to test the fix though. If this PR lands first I will create a follow up to adjust the timeout back down.